### PR TITLE
[bitnami/metrics-server] Implement `dnsPolicy`

### PR DESCRIPTION
**Pull request by notjames**
Signed-off-by: Jim Conner <snafu.x@gmail.com>

**Description of the change**

implements `dnsPolicy` for the bitnami `metrics-server` chart

**Benefits**

Given certain conditions of a cluster IE CNI, webhooks, etc, one may need to set the `dnsPolicy` to something other than `Default` in order for name resolution et al to work within pods.

**Possible drawbacks**

None of which I'm aware.

**Applicable issues**

N/A

**Additional information**
I wasn't able to proper test this chart change due 

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by pablogalegoc** [comment]
If you are facing issues solving the lint warnings you can download [ct](https://github.com/helm/chart-testing) and run `ct lint --config .github/ct-lint.yaml`, or let me know if you want me to take this PR and solve these issues

File: bitnami/metrics-server/values.yaml [hunk 1]
```diff
@@ -120,6 +120,10 @@ securePort: 8443
 ## if you use Weave network on EKS
 ##
 hostNetwork: false
+## @param dnsPolicy Default dnsPolicy setting
+## If you enable hostNetwork then you may need to set your dnsPolicy to something other
+## than "Default" depending on your requirements.
```
[the remark this answers is not in the document]
Comment (pablogalegoc) [reply bitnami/metrics-server/values.yaml:125]: @notjames these comments were mark as resolved but the changes were not applied, is there a reason for it?

**Comment by notjames** [comment]
> If you are facing issues solving the lint warnings you can download [ct](https://github.com/helm/chart-testing) and run `ct lint --config .github/ct-lint.yaml`, or let me know if you want me to take this PR and solve these issues

Ah! Yes, perfect. Lemme give this a try, because I was having issues and wasn't aware of this `ct` tool. More to come.

**Comment by notjames** [comment]
> If you are facing issues solving the lint warnings you can download [ct](https://github.com/helm/chart-testing) and run `ct lint --config .github/ct-lint.yaml`, or let me know if you want me to take this PR and solve these issues

When running the `ct` command, I'm getting output that indicates `ct` is trying to use `helm` < v3.x.
```
>>> helm init --client-only
Error: unknown command "init" for "helm"

Did you mean this?
        lint

Run 'helm --help' for usage.
Error linting charts: Error initializing Helm: Error waiting for process: exit status 1
------------------------------------------------------------------------------------------------------------------------
No chart changes detected.
------------------------------------------------------------------------------------------------------------------------
```

I'll give this a few more tries and if I can't figure this out, I'll defer to your kind offer to resolve the PR issues.

**Comment by notjames** [comment]
> > If you are facing issues solving the lint warnings you can download [ct](https://github.com/helm/chart-testing) and run `ct lint --config .github/ct-lint.yaml`, or let me know if you want me to take this PR and solve these issues
> 
> When running the `ct` command, I'm getting output that indicates `ct` is trying to use `helm` < v3.x.
> 
> ```
> >>> helm init --client-only
> Error: unknown command "init" for "helm"
> 
> Did you mean this?
>         lint
> 
> Run 'helm --help' for usage.
> Error linting charts: Error initializing Helm: Error waiting for process: exit status 1
> ------------------------------------------------------------------------------------------------------------------------
> No chart changes detected.
> ------------------------------------------------------------------------------------------------------------------------
> ```
> 
> I'll give this a few more tries and if I can't figure this out, I'll defer to your kind offer to resolve the PR issues.

Yup, I can't get past the helm issue and have been oncall this week so getting time to figure this out has been slim. If you wouldn't mind, would you please solve the issues?

**Comment by pablogalegoc** [comment]
Done! Thanks for the work @notjames. I hope that for the next contribution you don't face these issues.

**Comment by notjames** [comment]
> Done! Thanks for the work @notjames. I hope that for the next contribution you don't face these issues.

Thank you!

**Pull request merged**